src/modules/focusable.js

```diff
@@ -118,3 +125,25 @@ function makeDefaultHandler( id ) {
 		send( 'control-focus', id );
 	};
 }
+
+function createPartialUpdateHandler( elements ) {
+	return ( partial ) => {
+		const elementsToUpdate = elements.filter( element => {
+			const $container = element.$partialContainer;
+
+			if ( ! $container || $container.data( 'customize-partial-id' ) !== partial.id ) {
+				return false;
+			}
+
+			if ( 'function' === typeof element.onPartialUpdate ) {
+				element.onPartialUpdate.call( element, partial, element, elements );
```

Comment:
The statement just calls the callback if it exists, and the callback can do anything including setting the element's target.
Hmm... I don't get it what you concerned. Can you give me more detail?